### ISSUE TITLE
[Bug]: Null Pointer Exception during bootup

### ISSUE THREAD

**Issue by Devilswarchild** [Brian-Wuest/MC-Prefab#293]
### Minecraft Version

1.21.1

### Forge Version

21.1.69

### Mod Version

prefab-neoforge-1.o.o.jar

### Details

not sure how to describe, just the error I get when trying to launch the build of my modpack for 1.21

[crashreport](https://mclo.gs/3d5IqOr)

**Comment by Brian-Wuest** [comment]
Please provide a link to your modpack in curseforge or modernity for duplication.

If this is a private pack, please produce the full list of mods as well as versions as it might be due to other mods or the version of Neoforge.

Have you tried using Neoforge 21.1.62 and seeing if the issue goes away?

**Comment by Devilswarchild** [comment]
I have and issue persists

https://www.curseforge.com/minecraft/modpacks/minecolonies-dimensional-adventure 1.21.1 build is an alpha currently

**Comment by Brian-Wuest** [comment]
I was able to get your modpack to start a world though I had to disable a different mod.

`almostunified` was causing crashes for me so I disabled it even though Prefab was disabled so I disabled that and I was able to start a world. I then enabled prefab and was able to play no problem.

Some screen shots of what I disabled as well as from the game

![image](https://github.com/user-attachments/assets/262760df-5248-4601-b827-13c5f901220b)


![image](https://github.com/user-attachments/assets/583d26dd-0bbc-4821-b6d6-918f121eb23d)

![image](https://github.com/user-attachments/assets/31186939-5f9d-4160-a970-dec5e2e696ff)


Since this isn't an issue with Prefab and cannot be duplicated, closing issue.
